.hlint.yaml

```diff
@@ -18,6 +18,7 @@
 - ignore: {name: "Avoid lambda using `infix`"}
 - ignore: {name: "Fuse mapM/map"}
 - ignore: {name: "Eta reduce"} # This warning will often make suggestions that are no longer valid due to simplified subsumption
+- ignore: {name: "Redundant <$>"}
```

Comment:
If I understand correctly, this is included for better clarity on the AST traversals, right?

Reply:
Yup.